test/tests/postprocessors/interface_value/interface_integral_variable_value_postprocessor_fv.i

```diff
@@ -0,0 +1,169 @@
+[Mesh]
+  [gen]
+    type = GeneratedMeshGenerator
+    dim = 2
+    nx = 6
+    xmax = 3
+    ny = 9
+    ymax = 3
+    elem_type = QUAD4
+  []
+  [./subdomain_id]
+    input = gen
+    type = SubdomainBoundingBoxGenerator
+    bottom_left = '0 0 0'
+    top_right = '2 1 0'
+    block_id = 1
+    [../]
+  [./interface]
+    type = SideSetsBetweenSubdomainsGenerator
+    input = subdomain_id
+    primary_block = '0'
+    paired_block = '1'
+    new_boundary = 'interface'
+  [../]
+[]
+
+[Functions]
+  [./fn_exact]
+    type = ParsedFunction
+    value = 'x*x+y*y'
+  [../]
+
+  [./ffn]
+    type = ParsedFunction
+    value = -4
+  [../]
+[]
+
+[Variables]
+  [./u]
+    family = MONOMIAL
+    order = CONSTANT
+    fv = true
+  [../]
+[]
+
+[FVKernels]
+  [./diff]
+    type = FVDiffusion
+    variable = u
+    coeff = 1
+  [../]
+
+  [./ffn]
+    type = FVBodyForce
+    variable = u
+    function = ffn
+  [../]
+[]
+
+[FVBCs]
+  [./all]
+    type = FVFunctionDirichletBC
+    variable = u
+    boundary = '0 1 2 3'
+    function = fn_exact
+  [../]
+[]
+
+[Materials]
+  [./stateful1]
+    type = StatefulMaterial
+    block = 0
+    initial_diffusivity = 5
+  [../]
+  [./stateful2]
+    type = StatefulMaterial
+    block = 1
+    initial_diffusivity = 2
+  [../]
+[]
+
+[AuxKernels]
+  [./diffusivity_1]
+    type = MaterialRealAux
+    property = diffusivity
+    variable = diffusivity_1
+  []
+  [./diffusivity_2]
+    type = MaterialRealAux
+    property = diffusivity
+    variable = diffusivity_2
+  []
+[]
+
+[AuxVariables]
+  [./diffusivity_1]
+    family = MONOMIAL
+    order = CONSTANT
+  []
+  [./diffusivity_2]
+    family = MONOMIAL
+    order = CONSTANT
+  []
+[]
+
+[Postprocessors]
+  [./diffusivity_average]
+    type = InterfaceIntegralVariableValuePostprocessor
+    interface_value_type = average
+    variable = diffusivity_1
+    neighbor_variable = diffusivity_2
+    execute_on = TIMESTEP_END
+    boundary = 'interface'
+  [../]
+  [./diffusivity_jump_primary_secondary]
+    type = InterfaceIntegralVariableValuePostprocessor
+    interface_value_type = jump_primary_minus_secondary
+    variable = diffusivity_1
+    neighbor_variable = diffusivity_2
+    execute_on = TIMESTEP_END
+    boundary = 'interface'
+  [../]
+  [./diffusivity_jump_secondary_primary]
+    type = InterfaceIntegralVariableValuePostprocessor
+    interface_value_type = jump_secondary_minus_primary
+    variable = diffusivity_1
+    neighbor_variable = diffusivity_2
```

Comment:
I think this input file can be combined with the average variable value postprocessor one.  Just do `type=${pp_type}` and then have a top-level `pp_type = foo` parameter that you then set in the test spec file via the `cli_args` param to be either InterfaceIntegralVariableValuePostprocessor or InterfaceAverageVariableValuePostrocessor.

Reply:
ok done.
I also did the same for the FE tests for consistency. Two less input files to download!